.gitignore

```diff
@@ -94,3 +94,5 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+.idea
```

Comment:
What is rope project? Why is .idea needed?

Reply:
Hi @hamishwillee!

This is manual fixing according to PEP8(https://www.python.org/dev/peps/pep-0008/#blank-lines).
Yes, I was tested and all working is fine with PyCharm IDE(https://www.jetbrains.com/pycharm/).
.idea not needed, this my local directory. When I create some project on my local machine with PyCharm, this directory .idea created.

Thank you!

Reply:
Yes, it's done.
Thank you!

Reply:
Thank you. I'll do quick test and merge shortly (probably tomorrow).